src/@types/express.ext.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace Express {
+  export interface Response {
```

Comment:
i don't really understand what's happening here, could we just have safeRedirect as a standalone function that can be imported when needed.

Reply:
Here I just extend Response object add new method `safeRedirect`
i created standalone function, but this approach better.